easybuild/tools/options.py

```diff
@@ -681,7 +692,7 @@ def process_software_build_specs(options):
         tryval = getattr(options, 'try_%s' % opt)
         if val or tryval:
             if val and tryval:
-                self.log.warning("Ignoring --try-%(opt)s, only using --%(opt)s specification" % {'opt': opt})
+                logger.warning("Ignoring --try-%(opt)s, only using --%(opt)s specification" % {'opt': opt})
```

Comment:
We don't have 100% coverage, so some things are still not tested: https://jenkins1.ugent.be/view/EasyBuild/job/easybuild-framework_unit-test_hpcugent_master/Coverage/

However, these are usually hard to reach places in the code, so should have very low user impact, I would only put to much effort in trying to have a unit test for them after users actually run into them. (getting 100% coverage should not be a goal in itself)